website/docs/concepts/scopes.mdx

```diff
@@ -0,0 +1,331 @@
+---
+title: Scopes
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Scoping in Riverpod is a very powerful feature, but like all powerful features, it should be used wisely and intentionally.
+
+Scoping allows you to create new instances of the same provider in different parts of the tree, just like `package:provider` does.
```

Comment:
"create new instances of the same provider" might be unclear

I would instead say "Scoping allows the same provider to point to a different state in different parts of the widget tree. It is similar to how Flutter exposes a `Theme` for the entire application, yet it is possible to override that `Theme` for a specific widget without impacting other widgets."